pages/index.html

```diff
@@ -5,8 +5,46 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+  <section class = "container">
+  <header>
+    <div class = "title"><h1>Personal Portfolio - Stacy Ji</h1></div>
+    <div class = "link"><nav>
+        <ul>
+            <li>
+                <a href = "portfolio.html"> Portfolio</a>
+            </li>
+            <li>
+                <a href = "about.html"> About</a>
+            </li>
+        </ul>
+    </nav>
+  </div>
+</header>
+</section>
+<main>
+  <section class="short-bio">
+    <p>Hey y'all! My name is Stacy Ji. I am a student of Ada Developers Academy and a software Developer! I am part of Tiger Class. </p>
+    <p>I enjoy coding, I would like to be a coder for the rest of my life!</p>
+  </section>
+</main>
+
+<footer>
+  <nav>
+    <ul>
+      <li>
+        <a href = "#"> About me </a>
+      </li>
+      <li>
+        <a href = "#"> Contact </a>
```

Comment:
It is good practice to have your links on every page. That way a user can easily change between pages on any page.